packages/e2e-tests/helpers/utils.ts

```diff
@@ -84,6 +84,25 @@ export async function mapLocators<T>(
   );
 }
 
+export async function waitForRecordingToFinishIndexing(page: Page): Promise<void> {
```

Comment:
Using this new helper method made my tests a lot more stable when the backend was being slow today. Maybe we should consider doing this in more (or every?) test.